runners/spark/src/test/java/org/apache/beam/runners/spark/translation/streaming/FlattenStreamingTest.java

```diff
@@ -80,4 +90,35 @@ public void testRun() throws Exception {
     res.close();
   }
 
+  @Test
+  public void testFlattenBoundedUnbounded() throws Exception {
```

Comment:
These tests really should be added to `FlattenTest` in the main SDK as `RunnableOnService`, as soon as that is feasible and can be invoked properly here. What is needed? I think it means the runner needs to watch the `PAssert` aggregators (of passes/fails) and also have a timeout.